spec/fixtures/bosh-lite/cf-manifest.yml

```diff
@@ -7,8 +7,8 @@ compilation:
 director_uuid: DIRECTOR_UUID
 jobs:
 - default_networks:
-    - name: cf1
-      static_ips: null
```

Comment:
This is valid yaml, but we (the team prefer arrays to be directly under the parent key rather than tabbed in)